### PR TITLE
New App: Coinbase

### PR DESCRIPTION
**Pull request by harrywynn**
Displays your current Coinbase balance and holdings.

coinbase

### CODE REVIEW

**Comment by harrywynn** [comment]
I have read the CLA Document and I hereby sign the CLA

File: apps/coinbase/coinbase.star [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+COINBASE_CLIENT_SECRET = secret.decrypt("AV6+xWcEJMC6gX7GH0hewzGQ/B/2l0jpAtMyD1BZxfZLcL2vF9ZPHsaL+/TK9l7tDVs700mxxmJERtbBtGwLWE2RQPY8KdF88yWsYABohFu7GgfqjQ5LX0pqv6wrTqWfESyA7fPk7c3fgqOz3Hrr6AUrkddIDJuWpeZ8wywhZTYJad7q/Pd6RqsNVgX9qvhzOoFODy9O83rMM8GlKCZS6VHcIZG5DQ==")
+COINBASE_CLIENT_ID = secret.decrypt("AV6+xWcE2JaB2O2ordBy0T1hLm/fb3TSEfrEpGojMFF9kazLU04l+4czjZcjg10VMPjMyr884buJOXY070nmqgiP55bdN7ZtT484avoKQqoF8aM3VnE4Y1Jw//51a36IEk2/uJBmqGfY90H7QlYBQ76zVAL5DWkuVLGv6fI7Ux12Y7tv02gmWN77L+Hq+/GrfuV5I9c0wwM4nj5t/Zu1BIwrn3WPAQ==")
```
Comment (rohansingh) [inline apps/coinbase/coinbase.star:19]: Please use this client ID and secret:

```suggestion
COINBASE_CLIENT_SECRET = secret.decrypt("AV6+xWcEIOP/Rql2nyueyFjL9f51E2W1wDvcqtKyvXWRuSZwf7pNagbGLNridKAHG4Nw4oW2FZssbHNfsGwWAmSqXG3VN8oSvw0UY+4AB2LU3HMl5eEN9A139V08wU3/vOX7ouCUtHWwNkhHRngkQHqQiZSTK0KNOYTaIO9aeb7uzFwLdMdATDEQ2ypUjrvOt4l8UM1HIpXaIjn8T5bE+q7AYgaLww==")
COINBASE_CLIENT_ID = secret.decrypt("AV6+xWcEVIfI5lKs4wUCRh+CyWiA2VJ8Jngv8g/klexY7x7qR4KGIopsbZOIq/syABtfSToDA/nx5J8Mhy+XakIUjxESmr3V9fKOeymlJd7G/0VImVbAJYoZoTU2PMgzHcr7+HUV3SVUGrmZk62eMH5y77pWPb6MXIeLXAwTYFL6ZBk3NQCkP6EIN0dOP9h+ubvvBWMLE1U3Imc9ZdqxC4XPEQmPPg==")
```

**Review by rohansingh — CHANGES_REQUESTED** [review]
The cache is global and shared between all users. Please include a hash of the auth token in the cache key to scope it to the user.

**Review by rohansingh — APPROVED** [review]

**Comment by rohansingh** [comment]
This is live now. Thanks for the contribution!

**Pull request merged**